### ISSUE TITLE
Smooth animations

### ISSUE THREAD

**Issue by ahuang11** [holoviz/holoviews#4571]
Is there a way to make smooth animations like https://github.com/NicholasARossi/Easing-Animations-with-Python
https://github.com/NicholasARossi/Easing-Animations-with-Python/raw/master/media/chroma.gif

**Comment by ahuang11** [comment]
Maybe hvplot?

**Comment by jlstevens** [comment]
Honestly I think this feature is out of the scope of holoviews. Happy to move this issue to `hvplot` as suggested if @philippjfr agrees it is in scope there.

**Comment by philippjfr** [comment]
Even less in scope there than here imo, hvPlot is explicitly concerned with plotting real data, while these transitions are more suited for other animations. In the end I'd really prefer it if Bokeh and/or Plotly implemented transitions between states in JS like other JS based plotting libraries.

**Comment by jlstevens** [comment]
I agree: ideally this would be implemented at the plotting library level.

**Comment by jonmmease** [comment]
Cross reference: Plotly.js has basic support for animating scatter and bar traces https://plotly.com/python/animations/.

One thing that would probably be in scope for the Plotly backend would be to have an option to enable smooth transitions for HoloMaps/DynamicMaps that return supported elements.

**Comment by jbednar** [comment]
That would be cool.  Everyone wants bar chart races! :-)

**Comment by MarcSkovMadsen** [comment]
I guess it will take some time before Bokeh or Plotly supports as awesome transitions as this.

![echarts2](https://user-images.githubusercontent.com/42288570/94691608-cba80880-0331-11eb-83a1-f85bd4fbb2e2.gif)

**Comment by jbednar** [comment]
Those animations seem like the easy case, with each intermediate stage being a valid Bokeh or Plotly plot.  I would think we could do that in HoloViews if we wanted.  Whereas bar chart races need the columns to swap order left to right, animating the transition so that you can see where the bar is moving to and where it came from, which is something that the plotting library would have to do because there is no valid chart with the bar in intermediate locations in that case.
